ui/pages/confirmation/templates/create-snap-account.js

```diff
@@ -0,0 +1,26 @@
+function getValues(pendingApproval, t, actions, _history) {
```

Comment:
We try and add unit tests for each template just for stability and to help coverage, usually just a snapshot test to ensure the visual is what we expect. The same would apply for the remove template also.